cpp/src/Slice/Parser.cpp

```diff
@@ -4555,19 +4551,25 @@ Slice::DataMember::DataMember(
 // ----------------------------------------------------------------------
```

Comment:
All the code below this is to allow the compilers to inject their `languagePrefix` into `libSlice`.